helpers/env-variables.ts

```diff
@@ -242,6 +242,21 @@ const getModelEnvs = (modelConfig: ModelConfig): EnvVar[] => {
           },
         ]
       : []),
+    ...(modelConfig.provider === "t-systems"
+      ? [
+          {
+            name: "T_SYSTEMS_LLMHUB_BASE_URL",
+            description:
+              "The base URL for the T-Systems LLM Hub API. Eg: http://localhost:11434",
+            value: modelConfig.apiBase,
```

Comment:
as you never ask for this value, just set the default here and remove `apiBase`
```suggestion
            value: LLMHUB_API_URL
```